README.md

```diff
@@ -63,6 +63,29 @@
   ./spin.py destroy [dev|stage]
 ~~~
 
-#### Notes
+
 
 -   Do not use vagrant directly to spin and environment. It might work, but most likely it won't
+==============================================================================================
+Steps for Docker:
+Setting the App 
+ 1. install docker or docker for windows 
+ 2. Run  in the directory where docker file is located:
+ "docker build -t spincontainer ."
+
+ 3. Run 
+ "docker run -t -d --name spincontainer -v /var/run/docker.sock:/var/run/docker.sock spinner"
```

Comment:
docker run -t -d --name spincontainer -v /var/run/docker.sock:/var/run/docker.sock spinner
Unable to find image 'spinner:latest' locally
docker: Error response from daemon: pull access denied for spinner, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.